torchaudio/csrc/ffmpeg/stream_reader/stream_processor.cpp

```diff
@@ -82,7 +84,11 @@ int StreamProcessor::process_packet(AVPacket* packet) {
       return send_frame(NULL);
     if (ret < 0)
       return ret;
-    send_frame(pFrame1);
+
+    if (pFrame1->pts >= discard_before_pts)
+      send_frame(pFrame1);
```

Comment:
Please use parenthesis even if it is one-line. FBcode linter will complain later.